.env.example

```diff
@@ -68,6 +69,12 @@ REDISEARCH_PUBLISH_PORT=16379
 REDISEARCH_SLAVE_PUBLISH_PORT=16380
 POSTGRES_PUBLISH_PORT=5432
 ZOOKEEPER_PUBLISH_PORT=2181
+COORDINATOR_PUBLISH_PORT=8082
+BROKER_PUBLISH_PORT=8083
+HISTORICAL_PUBLISH_PORT=8084
+MIDDLEMANAGER_PUBLISH_PORT=8091
+MIDDLEMANAGER_PUBLISH_PORT_RANGE='8100-8105'
+ROUTER_PUBLISH_PORT=8888
```

Comment:
Please add prefix `APACHE_DRUID_` before these envs to identify them easily.

Reply:
Prefix `APACHE_DRUID_` added.